### PR TITLE
fix: breadcrumb last item color does not use the `breadcrumbLastItemColor` token.

### PR DESCRIPTION
**Pull request by messaooudi**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### Problem
The Breadcrumb last item text color does not follow the `colorText` seed token.
image

### Solution
The Breadcrumb `li:last-child` color CSS property should be set to `token.breadcrumbLastItemColor`.
image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix Breadcrumb last item color.          |
| 🇨🇳 Chinese | 修复 Breadcrumb 最后一项颜色。           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by MadCcc** [comment]
Changelog plz。

File: components/breadcrumb/style/index.ts [hunk 1]
```diff
@@ -52,8 +52,12 @@ const genBreadcrumbStyle: GenerateStyle<BreadcrumbToken, CSSObject> = (token) =>
         ...genFocusStyle(token),
       },
 
-      [`li:last-child > ${componentCls}-separator`]: {
-        display: 'none',
+      [`li:last-child`]: {
+        color: token.breadcrumbLastItemColor,
+
+        [`${componentCls}-separator`]: {
```
Comment (MadCcc) [inline components/breadcrumb/style/index.ts:58]:
```suggestion
        [`& > ${componentCls}-separator`]: {
```

**Comment by messaooudi** [comment]
> Changelog plz.

@MadCcc Thank you for your time, I added a change log entry and as a result the PR was automatically closed, I'm assuming that I did not have to do it in the change log file !

**Comment by afc163** [comment]
图片

You can write changelog here, instead of updating changelog.md files.

**Comment by messaooudi** [comment]
> 图片
> 
> You can write changelog here, instead of updating changelog.md files.

Thanks, should I reopen this PR and revert the change log commit ?

**Review by afc163 — APPROVED** [review]

**Pull request merged**